package.json

```diff
@@ -30,6 +30,7 @@
     "lint-staged": {
         "*.{js,json,md,yml}": [
             "prettier --write",
+            "yarn test",
```

Comment:
Hrm. I don't know about adding this here. To my knowledge, `lint-staged` gets called per file and passes that file path to the commands here. That means we'd be running `yarn test` and thus `yarn build && yarn lint && yarn coverage` over the _entire project_ for _each file_ staged because the `yarn test` command isn't setup in a way to run on a per file basis.